rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -331,7 +334,22 @@ def on_node_executor_start(cls):
         cls.agent_server_client = get_agent_server_client()
 
     @classmethod
+    @error_logged
     def on_node_execution(cls, q: ExecutionQueue[NodeExecution], data: NodeExecution):
+        timing_info, _ = cls.__on_node_execution(q, data)
+        cls.loop.run_until_complete(
+            update_node_execution_stats(
+                data.node_exec_id,
+                {
+                    "wall_time": timing_info.wall_time,
+                    "cpu_time": timing_info.cpu_time,
+                },
+            )
+        )
+
+    @classmethod
+    @time_measured
+    def __on_node_execution(cls, q: ExecutionQueue[NodeExecution], data: NodeExecution):
```

Comment:
Why double underscore?

Reply:
It's the internal helper method for `def on_node_execution`
Is one underscore the right convention?

Reply:
updated to one underscore